### DISCUSSION TITLE
Monthly ESMValTool community meeting - January 2024

### DISCUSSION THREAD

**Discussion by paultgriffiths** [ESMValGroup/Community#142] [Meetings]
The next online community meeting will be on

January 22nd, 2024 at 2pm UTC / 3pm CET

The duration will be one hour.[Join the meeting](https://ncas.zoom.us/j/98313515921?pwd=R2RIZGk4bXYzMW91anU1WUYzNjNEdz09)

Calendar invitations to the meeting are sent to the mailing list, join it [here.](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list)

**Format and agenda**

Anyone with an interest in ESMValTool is welcome to join and ask questions

We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.

Using the input from the introduction round, we choose some topics to talk about in more detail.

If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.

GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in #138

**Comment by malininae** [comment]
https://github.com/ESMValGroup/ESMValCore/issues/2304 

I also would like some questions on policies on how to write tests for the core.

**Comment by glpotter** [comment]
@paultgriffiths  asked me to furnish motivation for cmorizing and distribution all of the major reanalyses. Here is something Mike Bosilovich and I put together several months ago to hopefully convince the NASA GSFC data center that it needs to be done. We based it on the CREATE service I led when I was at NASA (Potter et. al. 2018, Bull. Amer. Meteor. Soc., 99, 677–687, https://doi.org/10.1175/BAMS-D-17-0174.1) .  Here is our input on motivation:

The NASA Collaborative Reanalysis Technical Environment (CREATE, Potter et al. 2018) provided access to reanalysis data in one location in a standard format (one variable per file, and standardized meta-data in the CMIP5 style). The collection included monthly and 6-hourly data from the eight major atmospheric reanalyses: CFSR, ERA-5, ERA-Interim, MERRA, MERRA-2, JRA-25 , JRA-55, and 20CRv2c. Most of these assimilate NASA’s and international satellite radiances, yet come to a different result in the physics terms owing to inherent model error and bias.

Except for NASA’s MERRA and MERRA-2, each reanalysis required a different data retrieval method. In addition, ERA-interim, JRA-25, JRA-55, and CFSR all required conversion from GRIB to NetCDF. Each reanalysis also required a unique set of procedures (to consider resolution, frequency and variable conventions) to repackage the data into the standardized format.

Bosilovich et al. (2009) demonstrated that a number of errors in individual analyses are reduced when combined as an ensemble average, but reproduced well observations of monthly precipitation and outgoing long wave radiation. CREATE contains the ensemble mean of the individual reanalyses and standard deviation for each month of key fields from the atmospheric reanalyses. A preliminary view of the Multiple Reanalysis Ensemble (MRE) suggests a similar result for a few selected variables in most of the reanalyses used to make the ensemble mean. This suggests that the MRE will make a better reference data set for reanalysis development than comparison to any one of the other reanalyses. To perform the ensemble mean, it is necessary to regrid each reanalysis to a standard horizontal and vertical grid. In addition, units and variable names are also converted to a standard set.
One of the primary benefits for gathering multiple reanalyses onto one site, and formatting them similarly, is the ease with which a user can both access the data and analyze it with a single workflow. Since the data is organized by variable, it is a simple task to retrieve the same variable from several reanalyses and extract all or part of the time series. In addition, the format conversion adheres to the conventions used for CMIP climate model experiments, so that the MRE and member reanalyses are easily compared to the suite of climate simulations.

The current CREATE data holdings for the primary reanalyses are now out-of-date and only span time from 1980 to 2020. As time marches forward, new climate events demand attention and maintaining the MRE into future times will allow understanding of the current events in a changing climate. In addition, the computing platforms and software at NCCS have been updated
and no longer support the customized software needed to download and repackage the data using the current standardized requirements for distribution. We also anticipate new reanalyses developments in the next few years to continue the advancement of the quality of individual reanalyses (including JRA-3Q, ERA6 and NASA’s R21C and MERRA-3). While we anticipate updates to include new reanalyses into the MRE, in the production stage, the MRE would provide a superior baseline reference data collection.
The S-RIP project (Fujiwara et al. 2017) provides a considerable example of how this compilation can be used to research both the reanalyses systems and Earth system processes by using and intercomparing many reanalyses. We envision that CREATE 2.0 could facilitate further projects like this in many potential studies. There is an increasing need from the climate modeling and research communities to produce CREATE 2.0 that will contain new data sets that adhere to the newest data standards, will be updated regularly as new data becomes available, and will be distributed using the best available technology. The recent formation of the WCRP Earth System Modeling and Observations panel has highlighted the original CREATE work as a structural component of enabling researchers to intercompare processes in reanalyses.

**Comment by glpotter** [comment]
@k-a-webb  asked about model diagnostics for model developers: Here is a link to the LLNL effort:
 https://doi.org/10.5194/gmd-15-9031-2022

**Comment by paultgriffiths** [comment]
Minutes:

@glpotter discussing with Copernicus about using ESMValTool to CMOR-ise reanalysis.  Further discussions needed re:scope.  Action - collate what new science is facilitated by CMOR-ising higher frequency output.  CMOR-isation a good QA/QC activity.

@k-a-webb - interested in using ESMValTool for model development and traceability of modifications, baselining with respect to observations, developing/implementing standard diagnostics/standard plots, calling ESMValTool via reduced complexity interfaces.

@egalytska working with Fresh Eyes on CMIP, summarising available tools, trying to avoid overlap with other activities in other WGs.  Session at AOGS 🎉 

@malininae questions about how to test preprocessors which generate derived variables (e.g. RH), when the derivation is an approximation. Do tests need to be purely technical or scientific aspect needs to be covered.

**Reply by paultgriffiths** [reply]
@egalytska and @malininae - i've tried to capture your comments but have not really done them justice 😞  - please edit/amend!

**Reply by k-a-webb** [reply]
Commenting to provide a link to my contact details (@k-a-webb). Apologies Paul for providing the incorrect username!
